sql/type.go

```diff
@@ -875,6 +875,12 @@ func IsArray(t Type) bool {
 	return ok
 }
 
+// IsJSON returns whether the given type is a json.
+func IsJSON(t Type) bool {
```

Comment:
This function is not really needed. You can just `typ == sql.JSON`

Reply:
The only reason why I added this _silly_ function is to be consistent with `IsArray` in `ArrayLength.Eval`. 
Anyway, it's already removed.

Reply:
Array is a type constructor, not a type itself, so you cannot do `typ == sql.Array`, that's why `IsArray` exists.